app/uk/gov/hmrc/fileupload/ApplicationLoader.scala

```
@@ -93,6 +95,16 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
     envelopeRepository.update() _
   }
 
+  def envelopeConstraintsConfiguration(config: Configuration) = {
+    EnvelopeConstraints(config.getInt("envelopeDefaultConstraints.maxNumFiles").getOrElse(throwRuntimeException("maxNumFiles")),
+                               config.getString("envelopeDefaultConstraints.maxSize").getOrElse(throwRuntimeException("maxSize")),
+                               config.getString("envelopeDefaultConstraints.maxSizePerItem").getOrElse(throwRuntimeException("maxSizePerItem")))
+  }
+
+  def throwRuntimeException(key: String) = {
```

all the constraints will throw this exception in case of unspecified constraint(s) instead of providing a default one?